src/libteam/patch/0016-block-retry-count-changes.patch

```diff
@@ -0,0 +1,141 @@
+Don't reset the retry count after setting it for 60 seconds
+
+From: Saikrishna Arcot <sarcot@microsoft.com>
+Date: 2023-01-18 14:26:36 -0800
+
+After setting the retry count to some custom value, if a normal LACP
+packet comes in without a custom retry count, don't reset it back to
```

Comment:
Is this description correct? Should this be `don't reset the retry count back to old/default retry count`?

Reply:
Fixed.